tutorials/simulators/OpenFOAM.md

```diff
@@ -61,12 +43,29 @@ tutorial using the OpenFOAM Foundation distribution.
 :language: python
 ```
 
-## Example Code - ESI Distribution
+The current example is devided into four steps:
+
+1. **Machine Configuration**: In this step, we define the machine type and start
+it.
+
+2. **Download Input Files**: In this step, we download the input files from the
+inductiva bucket.
+
+3. **Picking the simulator**: In this step, we choose the simulator we want to
+use. In this case, we are using the OpenFOAM Foundation distribution.
+
+4. **Run the Simulation**: In this step, we run the simulation using the `run`
+method and specify the `bash_script` responsible for running the simulation.
+
+The last three lines we are just waiting for the simulation to finish, downloading
```

Comment:
The last three lines handle post-simulation tasks: waiting for the simulation to finish, downloading the outputs, and terminating the machine, in that order.